### PR TITLE
chore(get-glamor-classname): remove theme argument

### PR DESCRIPTION
**Pull request by tdeschryver**
**What**:  Removing the `theme` argument, see #219 
**Why**:  Because `theme` is available in `props`
**How**:  By removing the `theme` argument and running the tests till they were all green

**Checklist**:
- [ ] Documentation N/A
- [x] Tests
- [x] Ready to be merged 
- [x] Added myself to contributors table 

**BREAKING CHANGE**: 
theme argument is being removed.  
Before:  
`const MyDiv = glamorous.div((props, theme) => ({   fontSize: theme.main.fontSize, }))`
After: 
`const MyDiv = glamorous.div(({theme}) => ({   fontSize: theme.main.fontSize, }))`

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -410,6 +410,15 @@
...
         "doc"
       ]
+    },
+    {
+      "login": "tdeschryver",
+      "name": "tdeschryver",
+      "avatar_url": "https://avatars2.githubusercontent.com/u/28659384?v=3",
+      "profile": "https://github.com/tdeschryver",
+      "contributions": [
+        "code"
```
Comment (kentcdodds) [inline .all-contributorsrc]: Could add `test` here too :wink:

Comment (tdeschryver) [inline .all-contributorsrc]: Yea I figured it should go to the `next` branch :smile:

Comment (tdeschryver) [inline .all-contributorsrc]: OK done, I think I did it right. :fearful:

**Comment by kentcdodds** [comment]
Merged into `next`. Thanks again! :clap:

**Comment by tdeschryver** [comment]
Thank you for making this an awesome experience! 
I hope this will be the first of many!

**Pull request merged**